interface-definitions/interfaces_openvpn.xml.in

```diff
@@ -14,6 +14,133 @@
             <format>vtunN</format>
             <description>OpenVPN interface name</description>
           </valueHelp>
+          <childSpecification>
+            <requiredChildren>
+              <child>mode</child>
+            </requiredChildren>
+            <!-- OpenVPN client mode -->
+            <oneWayDependantChildren>
+              <dependants><descendant name="mode"><value>client</value></descendant></dependants>
+              <dependees>
+                <child>remote_host</child>
+              </dependees>
+            </oneWayDependantChildren>
+            <mutuallyExclusiveChildren>
+              <descendant name="mode"><value>client</value></descendant>
+              <child>local-port</child>
+              <child>local-host</child>
+              <descendant name="tls"><child>dh-params</child></descendant>
+            </mutuallyExclusiveChildren>
+            <!-- / OpenVPN client mode -->
+            <!-- OpenVPN site-to-site -->
+            <oneWayDependantChildren>
+              <dependants><descendant name="mode"><value>site-to-site</value></descendant></dependants>
+              <dependees>
+                <child>remote-host</child>
```

Comment:
Site-to-site mode doesn't actually require `remote-host`. One side of a site-to-site tunnel is a listener.